flagger/__main__.py

```diff
@@ -0,0 +1,14 @@
+import sys
+from flagger import queries, aws
+
+
+def main():
+    dry_run = "--dry-run" in sys.argv
```

Comment:
I think some more error-handling here might be good - maybe error and exit if there are arguments that aren't `--dry-run`?

Reply:
Total aside, but related: there's also [`optparse`](https://docs.python.org/3/library/optparse.html) for command line script option parsing that's built-in to the library, and [`click`](https://click.palletsprojects.com/en/7.x/) for even more power. :-)

However, aside from a `--dry-run` option and maybe another optional argument for specifying a semaphore to use, I can't think of other options we'd need, so those may be overkill, too.

Reply:
[`argparse`](https://docs.python.org/3/library/argparse.html#module-argparse) is preferred over `optparse` - `optparse` has been deprecated since 3.2, but yeah, I think both would be overkill here

Reply:
D'oh!  Should've read the docs a bit more closely...  goes to show how often I write command line apps in Python. :-)

Reply:
Ended up doing a simple sanity check rather than `argparse` since we just have one possible option.